### PR TITLE
Started work on basic Sweden stuff.

### PR DESCRIPTION
**Pull request by diblaze**
Very basic stuff added. Pushing this up so Sweden has a "foundation".

Changelog:
Added Sweden-Norway flags.
Added Denmark as integrated puppet.
Added new Swedish division names (Norgeförsvaret)
Sweden starts with two defense forces (against Socialist Norway) in Oslo.

### CODE REVIEW

**Comment by Belsima** [comment]
image Just delete this file in all honesty.

File: redflood/history/countries/SWE - Sweden.txt [hunk 1]
```diff
@@ -10,6 +10,21 @@ add_ideas = {
...
 }
 
+#DENMARK PUPPET
+if = {
+	limit = {
+		has_dlc = "Together for Victory"
+	}
+	set_autonomy = {
+		target = DEN
+		autonomous_state = autonomy_integrated_puppet
```
Comment (Belsima) [inline redflood/history/countries/SWE - Sweden.txt]: Should be regular puppet from all I can tell, not integrated.

File: redflood/localisation/Sweden_l_english.yml [hunk 2]
```diff
@@ -40,7 +40,9 @@ SWE_revolutionary_nationalism:0 "Country"
 SWE_revolutionary_nationalism_DEF:0 "Country"
 SWE_revolutionary_nationalism_ADJ:0 "Countryn"
 #PARTIES# 
-#LEADER DESCRITIONS# 
+#LEADER DESCRITIONS AND BOOKMARK# 
+POLITICS_PER_ALBIN_HANSSON_DESC:0 "Per Albin Hansson got elected as Prime Minister in the year 1932 when his party 'Social Democrats' won the Election. Ever since then, he has lead the country to greatness and helping Norway in their civil war and taking them in under the protection of a union with Sweden."
+SWEDEN_NORWAY_DESC:0 "During the Velikaya Voyna, Sweden has become a strong neutral country by refusing to participate in the war. Sweden refused to help Germany take over Finland, and signed the neutrality pact instead alongside with Norway and Denmark. Germany did not like this, and in a last ditch effort attacked Denmark. Sweden quickly responded by sending troops to Copenhagen and helping them fight against the Germans. However, the Danish government fell quickly, making Sweden install a Swedish government in Denmark and kicking out the socialists in the country. Sweden helped Norway fight off the socialist uprising, and in the end got into a union with Norway."
```
Comment (Belsima) [inline redflood/localisation/Sweden_l_english.yml]: Clarify the COMMIES attacked Denmark and not the empire. Use something more sophisticated than 'Germany did not like this'; perhaps 'Germany did not take to this kindly'.

>However, the Danish government fell quickly, making Sweden install a Swedish government in Denmark and kicking out the socialists in the country.

should be

>However, the Danish government fell quickly, forcing Sweden to install a Swedish government in Denmark to ensure stability and to repel the socialist invasion.

File: redflood/history/countries/SWE - Sweden.txt (hunk 1 above)
Comment (diblaze) [reply redflood/history/countries/SWE - Sweden.txt]: After discussing briefly on Discord, Denmark will instead be guaranteed by Sweden.
From the perspective of the lore, having Denmark as a puppet for almost 16 years is not realistic.

**Pull request merged**